config/locales/zho.yml

```diff
@@ -0,0 +1,114 @@
+zho:
+  assessments:
+    html:
+      shared:
+        email: '电子邮件'
+        phone: '电话'
+        webpage: '网页'
+      weblink:
+        title: '日常自我报告 '
+        bool-title: '请选择您正经历的所有症状'
+        instruction1: '如果您没有报告任何症状，请继续遵循由当地卫生部门提供的建议。'
+        instruction2: '如果您报告了任何症状，当地卫生部门将会尽快与您联系。如果您有任何紧急问题，请联系您的医疗服务提供者或当地卫生部门。避免与其他人密切接触，并留在家中。'
+        instruction3: '如果您遇到紧急医疗情况，请拨打911，并让他们知道你正在接受卫生部门的监控。'
+        instruction4: '请使用下面的联系信息与负责您病例的公共健康官员联系。'
+        submit: '提交'
+        thank_you: '感谢您完成您的自我报告'
+      email:
+        shared:
+          greeting: '亲爱的%{name},'
+          report: '每日报告'
+          footer: >-
+            不要回复此电子邮件，转发此电子邮件或分享此链接。此消息是由Sara Alert系统自动生成的，并且是唯一的，仅供您使用。如果您希望停止接收这些通知，或者认为这是一个错误，请联系您当地的公共卫生部门。
+            有关隐私和更多信息，请访问saraalert.org。
+        enrollment:
+          subject: '加入Sara Alert'
+          header: '欢迎使用Sara Alert系统'
+          info1: '您已加入Sara Alert监控系统。我们要求您提供您的每日状况报告。请单击下面的按钮，然后按照屏幕上的说明进行操作。'
+          info2: '您将每天收到了类似的提醒，直到你的监控期结束。'
+        reminder:
+          subject: 'Sara Alert报告提醒'
+          header: '这封邮件提醒您完成您的每日报告'
+          thank_you: '感谢您参加Sara Alert监控系统，请使用下面的链接填写您的每日报告。'
+        closed:
+          subject: 'Sara Alert监控期已结束'
+          header: '监控已完成！'
+          thank_you: 'Sara警报监控 %{initials_age}已于%{completed_date}完成！感谢您的参与。'
```

Comment:
I think this should just say `Sara Alert` in English right? We don't want to mix the `Sara` with the Chinese words for alert.

Reply:
This was the translation provided by the PO team, I'll check with them to see if they want this changed.